diff --git a/android/build.gradle b/android/build.gradle
index 53684692b..a54220356 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -137,7 +137,7 @@ allprojects {
             // Release every dependency the SDK has with a -jitsi-XXX qualified version. This allows
             // us to pin the dependencies and make sure they are always updated, no matter what.
 
-            project.version = "${json.version}-jitsi-${versionQualifierNumber}"
+            project.version = "${json.version}-cometchat-${versionQualifierNumber}"
 
             task jitsiAndroidSourcesJar(type: Jar) {
                 archiveClassifier = 'sources'
