diff --git a/android/build.gradle b/android/build.gradle
index 53684692b..a54220356 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -137,7 +137,7 @@ allprojects {
             // Release every dependency the SDK has with a -jitsi-XXX qualified version. This allows
             // us to pin the dependencies and make sure they are always updated, no matter what.
 
-            project.version = "${json.version}-jitsi-${versionQualifierNumber}"
+            project.version = "${json.version}-cometchat-${versionQualifierNumber}"
 
             task jitsiAndroidSourcesJar(type: Jar) {
                 archiveClassifier = 'sources'
diff --git a/android/scripts/release-sdk.sh b/android/scripts/release-sdk.sh
index 63536a9a2..82265814b 100755
--- a/android/scripts/release-sdk.sh
+++ b/android/scripts/release-sdk.sh
@@ -39,12 +39,12 @@ pushd ${THIS_DIR}/../
 popd
 
 # The artifacts are now on the Maven repo, commit them
-if [[ $MVN_HTTP == 0 ]]; then
-    pushd ${MVN_REPO_PATH}
-    git add -A .
-    git commit -m "Jitsi Meet SDK + dependencies: ${SDK_VERSION}"
-    popd
-fi
+# if [[ $MVN_HTTP == 0 ]]; then
+#     pushd ${MVN_REPO_PATH}
+#     git add -A .
+#     git commit -m "Jitsi Meet SDK + dependencies: ${SDK_VERSION}"
+#     popd
+# fi
 
 # Done!
 echo "Finished! Don't forget to push the tag and the Maven repo artifacts."
